packages/admin/src/common/BrevoConfigProvider.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+export interface BrevoConfig {
+    apiUrl: string;
+}
+
+const BrevoConfigContext = React.createContext<BrevoConfig | undefined>(undefined);
+
+interface BrevoConfigProviderProps {
+    value: BrevoConfig;
+}
+
+export const BrevoConfigProvider = ({ children, value }: React.PropsWithChildren<BrevoConfigProviderProps>) => {
+    return <BrevoConfigContext.Provider value={value}>{children}</BrevoConfigContext.Provider>;
+};
+
+export const useBrevoConfig = (): BrevoConfig => {
+    const context = React.useContext(BrevoConfigContext);
+    if (context === undefined) {
+        throw new Error("useBrevoConfig must be used within a BrevoConfigProvider");
+    }
+    return context;
+};
```

Comment:
as we already talked, I like this config and I think other variables like the scopeParts should also be defined there.
Or maybe even defining everything in the BrevoConfigProvider would make it easier in the application to use, instead of here: https://github.com/vivid-planet/comet-brevo-module/blob/main/demo/admin/src/Routes.tsx#L51